rotkehlchen/tests/api/test_historical_balances.py

```diff
@@ -981,3 +982,45 @@ def test_historical_price_cache_only_special_assets(
     result = assert_proper_sync_response_with_result(response)
     assert str(START_TS) in result['prices']
     assert result['prices'][str(START_TS)] == '1400'
+
+
+@pytest.mark.parametrize('should_mock_price_queries', [False])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+def test_manual_prices_override_external_data(
+        rotkehlchen_api_server: 'APIServer',
+        gnosis_inquirer: 'GnosisInquirer',
+        globaldb: 'GlobalDBHandler',
+) -> None:
+    """Manual prices should override external market data in historical queries."""
+    # Add some historical manual price entries for token->EUR
+    for i in range(3):
```

Comment:
```suggestion
    for idx in range(3):
```
we generally use `idx` rather than just `i` for indexes